fetch_response.go

```diff
@@ -31,7 +33,8 @@ type FetchResponseBlock struct {
 	HighWaterMarkOffset int64
 	LastStableOffset    int64
 	AbortedTransactions []*AbortedTransaction
-	Records             Records
```

Comment:
Technically it's a breaking change to remove this; could you leave it with a comment that it's deprecated, and just fill it in with the first set or something?

Reply:
Done, let me know if that's what you had in mind.